### PR TITLE
Upgrade all versions base on V4

### PR DESCRIPTION
**Pull request by SeverusYixin**
This PR related to #291

### CODE REVIEW

File: .github/workflows/auto-add-download-statistics.yml [hunk 1]
```diff
@@ -13,9 +13,9 @@ jobs:
       uses: actions/checkout@v2
     
     - name: Set up Python
-      uses: actions/setup-python@v2
+      uses: actions/setup-python@v4  # Updated to the latest version of the action
```
Comment (haesleinhuepf) [inline .github/workflows/auto-add-download-statistics.yml:16]: Why is this comment necessary??

**Review by haesleinhuepf — CHANGES_REQUESTED** [review]
Please remove the comments from your modifications. I do not understand, why they are necessary. Also in the future, please avoid those.

**Comment by SeverusYixin** [comment]
Hi @haesleinhuepf, you can review it again and then close it.

**Comment by haesleinhuepf** [comment]
Awesome, thanks!

**Comment by SeverusYixin** [comment]
You are welcome!

**Pull request merged**